lib/remap-lang/src/parser.rs

```diff
@@ -504,7 +693,7 @@ mod tests {
             (
                 // Regexes can't be parsed as part of a path
                 r#".foo = split(.foo, ./[aa]/)"#,
-                vec![" 1:21\n", "= expected path_segment"],
+                vec![" 1:22\n", "= expected not"],
```

Comment:
This error is somewhat cryptic (better error messages is still on the TODO list), but the change is caused because you could technically do `. / 5` (in reality you can't because for Vector, `.` will always point to a map, so we might introduce a custom error for this case).